_observing-your-data/trace/ta-dashboards.md

```diff
@@ -1,25 +1,143 @@
+
+### Trace analytics visualizations
+
+**Services** visualizations, such as a table or map, help you logically analyze service behavior and accuracy. The following visualizations can help you identify anomalies and errors:
+
+- **Services table**
+  - A RED indicator, along with connected upstream and downstream services and other actions, is indicated in each table column. The following image is an example **Services** table.
+
+  ![Services table]({{site.url}}{{site.baseurl}}/images/trace-analytics/services-table.png)
+    
+  - General-purpose filter selection is used for any field or filter composition. The following image shows this filter.
+
+  ![Services filter selection]({{site.url}}{{site.baseurl}}/images/trace-analytics/services-filter-selection.png)
+
+  - The **Services** throughput tooltip provides an at-a-glance overview of a service's incoming request trend for the past 24 hours. The following image shows an example tooltip.
+
+  ![Services throughput tooltip ]({{site.url}}{{site.baseurl}}/images/trace-analytics/service-throughput-tooltip.png)
+  
+  - The **Services** correlation dialog window provides an at-a-glance overview of a service's details, including its 24-hour throughput trend. You can use these details to analyze correlated logs or traces by filtering based on the `serviceName` field. The following image shows this window.
+ 
+  ![Services correlation dialog window]({{site.url}}{{site.baseurl}}/images/trace-analytics/single-service-correlation-dialog.png)
+
+  - The **Services** RED metrics dialog window provides an at-a-glance overview of a service's RED metrics indicators, including 24-hour error, duration, and throughput rate. The following image shows this window.
+ 
+  ![Services RED metrics for duration]({{site.url}}{{site.baseurl}}/images/trace-analytics/single-service-RED-metrics.png)
+
+  - The **Span details** dialog window provides a trace's details. You can use this information to further analyze a trace's elements, such as attributes and associated logs. The following image shows this window.
+
+  ![Services Span details dialog window]({{site.url}}{{site.baseurl}}/images/trace-analytics/span-details-fly-out.png)
+
+- **Service map**
+  - The **Service map** displays nodes, each representing a service. Node color indicates the RED indicator severity for that service and its dependencies. The following image shows a map.
+
+  ![Services map tooltip]({{site.url}}{{site.baseurl}}/images/trace-analytics/service-details-tooltip.png)
+  
+  - Selecting a node opens a detailed dialog window for that service. This interactive map visualizes service interconnections, helping identify problems by service, unlike dashboards that identify issues by operation. Sort by error rate or latency to pinpoint potential problem areas.
```

Comment:
At the end of the first sentence, should "service" be "node" or something like "for its associated service"?